build.log

```diff
@@ -0,0 +1,141 @@
+macOS arm64 ('15.7', ('', '', ''), 'arm64')
```

Comment:
Clearly we cannot merge log files into the repository. If you want to propose a PR with the goal of eventually getting it merged, please do not include any garbage in the patch set. You can post these as attachment to a comment instead.

This goes for all `*.log` files in this PR.